packages/urls/src/index.ts

```diff
@@ -9,6 +9,8 @@ export const CHANGE_NAME_SERVERS_FINDING_OUT_NEW_NS = `${ root }/domains/change-
 export const CONCIERGE_SUPPORT = `${ root }/concierge-support/`;
 export const CONTACT = `${ root }/contact/`;
 export const CALYPSO_CONTACT = '/help/contact';
+export const CALYPSO_HELP = '/help';
+export const CALYPSO_HELP_WITH_HELP_CENTER = CALYPSO_HELP + '?help_center=home';
```

Comment:
Typo: this should be `help-center`.
```suggestion
export const CALYPSO_HELP_WITH_HELP_CENTER = CALYPSO_HELP + '?help-center=home';
```